bundles/org.eclipse.passage.lic.net/src/org/eclipse/passage/lic/internal/net/i18n/NetMessages.properties

```diff
@@ -0,0 +1,2 @@
+LicensingServerCoordinates_settings_are_blank=%s cannot be blank
```

Comment:
And again

Reply:
Woha! Too early! Was not ready for review! )

Reply:
Please, have a look now